tests/Maintenance/Services/LogInfoRetrieverServiceTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Mundipagg\Core\Test\Maintenance\Services;
+
+use Mundipagg\Core\Maintenance\Services\LogInfoRetrieverService;
+use Mundipagg\Core\Test\Abstractions\AbstractSetupTest;
+
+class LogInfoRetrieverServiceTest extends AbstractSetupTest
+{
+    /**
+     * @var LogInfoRetrieverService
+     */
+    private $logInfoRetriever;
+
+    public function setUp()
+    {
+        parent::setUp();
+        $this->logInfoRetriever = new LogInfoRetrieverService();
+    }
+
+    public function testRetrieveInfo()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [testRetrieveInfo accesses the super-global variable $_SERVER.](https://app.codacy.com/manual/mundipagg/ecommerce-module-core/pullRequest?prid=4926694)